common/sagas/config/web3.ts

```diff
@@ -24,8 +25,13 @@ export function* initWeb3Node(): SagaIterator {
 }
 
 // unset web3 as the selected node if a non-web3 wallet has been selected
-export function* unsetWeb3NodeOnWalletEvent(action): SagaIterator {
+export function* unsetWeb3NodeOnWalletEvent(
+  action: ResetWalletAction | SetWalletAction
+): SagaIterator {
   const node = yield select(getNodeId);
+  if (action.type === WalletTypeKeys.WALLET_RESET) {
+    return;
+  }
```

Comment:
Why even `takeEvery(WalletTypeKeys.WALLET_RESET, unsetWeb3NodeOnWalletEvent)` if we just return?

Reply:
was not part of this PR but good catch, ill remove it anyway